### PR TITLE
Review application layout

### PR DESCRIPTION
**Pull request by marilynechan**
Review application layout & use our own styleguide in our styleguide documentation!

### CODE REVIEW

**Comment by wfortin** [comment]
LGTM

File: scss/components/navigation.scss [hunk 1]
```diff
@@ -24,20 +24,15 @@ $navigation-locked-color: $medium-grey;
     position: relative;
     z-index: 51;
 
-    float: left;
-    width: $navigation-width;
     height: calc(100vh - #{$header-height});
+    overflow: auto;
+    @include slim-scroll(#444444, #d3d3d3);
```
Comment (GermainBergeron) [inline scss/components/navigation.scss]: We should use variable for colors

**Comment by GermainBergeron** [comment]
:shipit:

File: scss/components/navigation.scss (hunk 1 above)
Comment (marilynechan) [inline scss/components/navigation.scss]: Done, I used our light-grey/dark-grey as default values in slim-scroll mixin, and dark-grey/medium-grey as the "reversed" slim-scroll in the navigation.

**Pull request merged**